cumulusci/tests/test_schema.py

```diff
@@ -13,14 +13,15 @@ class TestSchema:
     )
     def test_schema_validates(self, cumulusci_test_repo_root):
         from jsonschema import validate
+        from yaml import Loader
 
         schemapath = (
             cumulusci_test_repo_root / "cumulusci/schema/cumulusci.jsonschema.json"
         )
         with open(schemapath) as f:
             schema = json.load(f)
         with open(cumulusci_test_repo_root / "cumulusci.yml") as f:
-            data = yaml.load(f)
+            data = yaml.load(f, Loader=Loader)
```

Comment:
@prescod - It looks like a `Loader` is [now required](https://pyyaml.org/wiki/PyYAMLDocumentation) here? This seems quite silly to impose on end users.

Reply:
Shouldn't we be using `yaml.safe_load` here?

Reply:
This is just a test file but yeah, yaml.safe_load will probably fix the error.